package com.catshi.nuguri.exceptions

class AlreadyAttendedException: Throwable() {
}